test/gpu/octree/test_template_bfrs.cpp

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
Use the SPDX blurb instead?

Reply:
Thanks, @kunaltyagi for the comment! The shorter license info is very nice, and I added it.